tests/integration/test_basic.py

```diff
@@ -243,3 +244,167 @@ def test_publish_mk8s_mixed_status(self, mixed_status_tests_for_mk8s):
+        """Verify that moving a test case to a test suite (keeping the same 'test_id') will show execution results under
+        the old test cycle and under a new test cycle named after the suite for the "asc" CI environment.
+        """
+
+        # Setup
+        single_passing_test_for_asc.assert_invoke_zigzag()
+        original_parent_test_cycle = single_passing_test_for_asc.tests[0].qtest_parent_test_cycles[0].name
+
+        # Expectations
+        new_parent_test_cycle_name_exp = 'TestSuite'
+
+        # Test
+        single_passing_test_for_asc.tests[0].class_name = "tests.test_default.{}".format(new_parent_test_cycle_name_exp)
+        single_passing_test_for_asc.assert_invoke_zigzag(force_clean_up=False)
+        parent_test_cycles = single_passing_test_for_asc.tests[0].qtest_parent_test_cycles
+
+        # The qTest API returns test associated test runs from newest to oldest.
+        pytest.helpers.assert_qtest_property(parent_test_cycles[0],
+                                             'name',
+                                             new_parent_test_cycle_name_exp)
+
+        pytest.helpers.assert_qtest_property(parent_test_cycles[1],
+                                             'name',
+                                             original_parent_test_cycle)
+
+    # noinspection PyUnresolvedReferences
+    def test_move_test_case_to_suite_for_mk8s(self, single_passing_test_for_mk8s):
+        """Verify that moving a test case to a test suite (keeping the same 'test_id') will show execution results under
+        the old test cycle and under a new test cycle named after the suite for the "asc" CI environment.
+        """
+
+        # Setup
+        single_passing_test_for_mk8s.assert_invoke_zigzag()
+        original_parent_test_cycle = single_passing_test_for_mk8s.tests[0].qtest_parent_test_cycles[0].name
+
+        # Expectations
+        new_parent_test_cycle_name_exp = 'tests.test_default.TestSuite'
+
+        # Test
+        single_passing_test_for_mk8s.tests[0].class_name = new_parent_test_cycle_name_exp
+        single_passing_test_for_mk8s.assert_invoke_zigzag(force_clean_up=False)
+        parent_test_cycles = single_passing_test_for_mk8s.tests[0].qtest_parent_test_cycles
+
+        # The qTest API returns test associated test runs from newest to oldest.
+        pytest.helpers.assert_qtest_property(parent_test_cycles[0],
+                                             'name',
+                                             new_parent_test_cycle_name_exp)
+
+        pytest.helpers.assert_qtest_property(parent_test_cycles[1],
+                                             'name',
+                                             original_parent_test_cycle)
+
+    # noinspection PyUnresolvedReferences
+    def test_move_test_case_to_different_file_for_asc(self, single_passing_test_for_asc):
+        """Verify that moving a test case to a test suite (keeping the same 'test_id') will show execution results under
```

Comment:
<whining>
Lines that max out the 120 character limit cause me to have to horizontal scroll in GitHub. I would be nice if we standardized on a line length a bit shorter. Even 118 characters would address this.
<\whining>

Reply:
![stop whining!](https://media.giphy.com/media/l396InTOf9zsp2EO4/giphy.gif)